src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -29,6 +30,13 @@ public OrdersController(IOrderRepository orderRepository)
         [Route("/api/v1/organisations/{organisationId}/[controller]")]
         public async Task<ActionResult> GetAllAsync(Guid organisationId)
         {
+            var primaryOrganisationId = User.GetPrimaryOrganisationId();
+
+            if (primaryOrganisationId != organisationId.ToString())
```

Comment:
By changing the above type to a GUID I believe this would mean don't have to worry about case?

Reply:
oh nice, you're right